packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -110,29 +119,29 @@ export class AttestationsWrapper extends BaseWrapper<Attestations> {
     valueToInt
   )
 
-  /**
-   * @notice Returns the unselected attestation request for an identifier/account pair, if any.
-   * @param identifier Hash of the identifier.
-   * @param account Address of the account.
-   */
   getUnselectedRequest = proxyCall(
     this.contract.methods.getUnselectedRequest,
-    tupleParser(PhoneNumberUtils.getPhoneHash, (x: string) => x),
+    undefined,
     (res) => ({
       blockNumber: valueToInt(res[0]),
       attestationsRequested: valueToInt(res[1]),
       attestationRequestFeeToken: res[2],
     })
   )
 
+  /**
+   * @notice Waits for appropriate block numbers for before issuer can be selected
+   * @param identifier Attestation identifier (e.g. phone hash)
+   * @param account Address of the account
+   */
   waitForSelectingIssuers = async (
-    phoneNumber: string,
+    identifier: string,
```

Comment:
What do you think about making the salting the responsibility of the wrapper instead of the consumer? IMO it would allow consumers to just pass things through without having to manage the salting themselves

Reply:
Given the complex interplay between the app, the native libraries and the phone number privacy service, I'm not sure how that would be possible

Reply:
Maybe I'm misunderstanding, but can't `identifier: string` be replaced by `phoneNumber: string, salt?: string`?

Reply:
Ah, I thought you wanted the the wrapper to handle actually *getting* the salts.
We _could_ do that, I guess it's just a matter of taste. I rather a simpler, generic interface that actually matches what the contract does. Also nice that it doesn't assume the id is a phone number for when we support other types.

Reply:
Oh, also also we needed it this way for the phone number privacy service, which for matchmaking will handle hashed phone numbers but not salts

Reply:
Up to you. What do you think of having two methods for this? My understanding of having wrappers in the first place is to have them abstract transformation from the contract to the consumer in the first place, but @mcortesi would know better